packages/react-native-codegen/src/parsers/utils.js

```diff
@@ -4,7 +4,7 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @flow strict-local
```

Comment:
Had to update this because the `TypeDeclarationMap` import inside `packages/react-native-codegen/src/parsers/parsers-commons.js` was complaining about `dependencies of a '@flow strict' module must also be '@flow strict'!`